.github/workflows/spec.yml

```diff
@@ -0,0 +1,29 @@
+name: Specification
+
+on:
+  pull_request:
+    paths:
+      - "pipeline/minimmit/spec"
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
```

Comment:
nit: in other worfklows, we don't put `\n` in the job definition (only between jobs)